open_academy/models/open_academy_session.py

```diff
@@ -41,3 +42,9 @@ def _onchange_seats(self):
                     "message": "The number of attendees can not be greater than the seats available"
                 }
             }
+
+    @api.constrains("instructor_id", "attendee_ids")
+    def _check_instructor_in_attendee(self):
+        for record in self:
+            if record.instructor_id and record.instructor_id in record.attendee_ids:
+                raise ValidationError(_("A session's instructor can't be an attendee"))
```

Comment:
Why are you using the `_` method? this method is for translations. And you still haven't reached that part of the course 🤔